config/news/trivial.txt

```diff
@@ -0,0 +1,82 @@
+Giant Hairball Has Perfect Grammar But Rolls rr's Too Much, Linguists Say
+{{AFFECTED}} Phonebooks Print All Wrong Numbers; Results In 15 New Marriages
+Tajaran Burglar Spotted on {{AFFECTED}}, Mistaken For Dalmatian
+Gibson Gazette Updates Frequently Absurd, Poll Indicates
+Esoteric Verbosity Culminates In Communicative Ennui, {{AFFECTED}} Academics Note
+Taj Demand Longer Breaks, Cleaner Litter, Slower Mice
+Survey: 3 Out Of 5 Skrell Loathe Modern Art
+Skrell Scientist Discovers Gravity While Falling Down Stairs
+Boy Saves Tajaran From Tree on {{AFFECTED}}, Thousands Cheer
+Shipment Of Apples Overturns, {{AFFECTED}} Diner Offers Applesauce Special
+Spotted Owl Spotted on {{AFFECTED}}
+Humans Everywhere Agree: Purring Tajarans Are Happy Tajarans
+From The Desk Of Wise Guy Sammy: One Word In This Gazette Is Sdrawkcab
+From The Desk Of Wise Guy Sammy: It's Hard To Have Too Much Shelf Space
+From The Desk Of Wise Guy Sammy: Wine And Friendships Get Better With Age
+From The Desk Of Wise Guy Sammy: The Insides Of Golf Balls Are Mostly Rubber Bands
+From The Desk Of Wise Guy Sammy: You Don't Have To Fool All The People, Just The Right Ones
+From The Desk Of Wise Guy Sammy: If You Made The Mess, You Clean It Up
+From The Desk Of Wise Guy Sammy: It Is Easier To Get Forgiveness Than Permission
+From The Desk Of Wise Guy Sammy: Check Your Facts Before Making A Fool Of Yourself
+From The Desk Of Wise Guy Sammy: You Can't Outwait A Bureaucracy
+From The Desk Of Wise Guy Sammy: It's Better To Yield Right Of Way Than To Demand It
+From The Desk Of Wise Guy Sammy: A Person Who Likes Cats Can't Be All Bad
+From The Desk Of Wise Guy Sammy: Help Is The Sunny Side Of Control
+From The Desk Of Wise Guy Sammy: Two Points Determine A Straight Line
+From The Desk Of Wise Guy Sammy: Reading Improves The Mind And Lifts The Spirit
+From The Desk Of Wise Guy Sammy: Better To Aim High And Miss Then To Aim Low And Hit
+From The Desk Of Wise Guy Sammy: Meteors Often Strike The Same Place More Than Once
+Tommy B. Saif Sez: Look Both Ways Before Boarding The Shuttle
+Tommy B. Saif Sez: Hold On; Sudden Stops Sometimes Necessary
+Tommy B. Saif Sez: Keep Fingers Away From Moving Panels
+Tommy B. Saif Sez: No Left Turn, Except Shuttles
+Tommy B. Saif Sez: Return Seats And Trays To Their Proper Upright Position
+Tommy B. Saif Sez: Eating And Drinking In Docking Bays Is Prohibited
+Tommy B. Saif Sez: Accept No Substitutes, And Don't Be Fooled By Imitations
+Tommy B. Saif Sez: Do Not Remove This Tag Under Penalty Of Law
+Tommy B. Saif Sez: Always Mix Thoroughly When So Instructed
+Tommy B. Saif Sez: Try To Keep Six Month's Expenses In Reserve
+Tommy B. Saif Sez: Change Not Given Without Purchase
+Tommy B. Saif Sez: If You Break It, You Buy It
+Tommy B. Saif Sez: Reservations Must Be Cancelled 48 Hours Prior To Event To Obtain Refund
+Doughnuts: Is There Anything They Can't Do
+If Tin Whistles Are Made Of Tin, What Do They Make Foghorns Out Of?
```

Comment:
Are these supposed to have the first letter of ever word capitalized? It would make sense if they were headlines, although articles and prepositions probably shouldn't be capitalized regardless.